modules/deploy-local-quay-ipv6.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: PROCEDURE
+[id="deploy-local-quay-ipv6"]
+= Deploying a local instance of {productname} in IPv6
+
+After you have created a new dual-stack container network, you can deploy a local instance of {productname}.
+
+.Prerequisites
+
+* You have configured Redis, your PostgreSQL database, local image storage, and creating a `config.yaml` file with the desired settings.
+
+.Procedure
+
+. In your {productname} `config.yaml` file, set the `FEATURE_LISTEN_IP_VERSION` field to `IPv6`. For example:
++
+[source,yaml]
+----
+# ...
+FEATURE_LISTEN_IP_VERSION: IPv6
```

Comment:
I think we might want to give a dual-stack configuration example, which should be the most typical one would want.
In this case I had to specify explicitly IPv6 port mapping when starting the container.
```
-p "[::]:80:8080" -p "[::]:443:8443"
```

Reply:
Fixed.